### DISCUSSION TITLE
Repository Star Count Doesn't Update

### DISCUSSION THREAD

**Discussion by gusanmaz** [community/community#52097] [Repositories]
### Select Topic Area

Bug

### Body

I have a repository that is liked over 10 maybe 20 times. Interestingly I see people starring this repo but star count of the repo stays at same number:5 for a long time.

I have never experienced this kind of behavior. What could be the cause of this? Could it be a bug?

URL of the repository where this happens: https://github.com/gusanmaz/BMB212-Algorithms-22-23
Fork Count: 25, Star Count: 5 Ratio of these numbers alone doesn't seem normal!

Thanks!

**Comment by YasinDehfuli** [comment]
You can use this link to report you bug to github to solve your problem
